### ISSUE TITLE
Fiches EVENTS - Renseigner les dates d'un event organisé sur plusieurs jours non consécutifs

### ISSUE THREAD

**Issue by ChloeCruzel** [chemins-de-la-transition/semapps#530]
Il arrive que les évènements (principalement les formations) ont lieu sur plusieurs jours non consécutifs. 

Or PROBLEME => la durée affichée dans le bandeau de la fiche **indique la durée entre le 1er jour et le dernier jour** de la formation, ce qui **ne renseigne pas la durée exacte de l'évènement**. 

Exemple : une formation de permaculture composée de 3 jours de formation qui a lieu sur 3 samedis à la suite. La durée indiquée dans le bandeau sera alors 3 semaines alors qu'en réalité la formation dure 3 jours. 

Serait-il possible de faire autrement ? 
Par exemple, la durée pourrait être renseignée à la main et non calculée automatiquement. 

Nous avons eu des retours de partenaires sur l'Occitanie qui trouvent cela confusant en l'état. 

Merci d'avance :)

**Comment by srosset81** [comment]
Effectivement, entrer la durée à la main serait le seul moyen de régler ce problème, car le serveur ne peut pas savoir qu'il y a des trous.

**Comment by GuillaumeAV** [comment]
Organize several events instead of one could be a solution : Helper text ?
